src/osd/OSD.cc

```diff
@@ -8061,6 +8063,16 @@ void OSD::handle_op(OpRequestRef& op, OSDMapRef& osdmap)
       return;
     }
 
+    // pool is full ?
+    pg_t _pgid = m->get_pg();
+    int64_t poolid = _pgid.pool();
+    const pg_pool_t *pi = osdmap->get_pg_pool(poolid);
```

Comment:
this may return NULL because the pool doesn't exist anymore/is bogus - in that case we can just return